src/openassetio-python/package/openassetio/managerApi/ManagerInterface.py

```diff
@@ -527,13 +552,21 @@ def getWithRelationship(
         host whether or not you are capable of handling queries for
         those relationships in this method.
         """
-        return [[] for _ in entityReferences]
+        for i in range(len(entityReferences)):
+            successCallback(i, [])
 
     def getWithRelationships(
-        self, relationshipTraitsDatas, entityReference, context, hostSession, resultTraitSet=None
+        self,
+        relationshipTraitsDatas,
+        entityReference,
+        context,
+        hostSession,
+        successCallback,
+        errorCallback,
+        resultTraitSet=None,
     ):
         """
-        Returns entity references that are related to the input
+        Queries entity references that are related to the input
         reference by the relationships defined by a set of traits and
         their properties.
```

Comment:
[Consider]
Sort of harping on the same point, but I wonder if defining the term `relationship` here to map to `relationshipTraitsDatas would help some readers apply the concept to the signature in the docs that come later in each method.
```suggestion
        Queries entity references that are related to the input
        reference by the relationships defined by a set of traits
        (\p relationshipTraitsDatas) and their properties.
```

N.B : is `and their properties` saying anything of value here?

Reply:
It was attempting to highlight that its a TraitsData - that can hold property values (vs a trait set), and that those property values influence the result of the query.

Reply:
Expanded on the topic [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/935/commits/d575d22518fb6a9167edd5f72ba3c57a77b8e4ea).

Reply:
Ty. I think just mentioning what argument we're talking about will help a lot.